Domains/4-Application/OpenSite/Released/OpenSiteDraft.01.00.02.ecschema.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- ==================================================================================
+|  * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+|  * See LICENSE.md in the project root for license terms and full copyright notice.
+======================================================================================= -->
+<ECSchema schemaName="OpenSiteDraft" alias="OpenSiteDraft" version="01.00.02" xmlns="http://www.bentley.com/schemas/Bentley.ECXML.3.2" displayLabel="OpenSite Draft" description="Drafts schema for drawing precise shapes in OpenSite+.">
+    <ECSchemaReference name="CoreCustomAttributes" version="01.00.03" alias="CoreCA" />
+    <ECSchemaReference name="BisCustomAttributes" version="01.00.00" alias="bisCA"/>
+    <ECSchemaReference name="BisCore" version="01.00.10" alias="bis" />
+    <ECSchemaReference name="RoadRailUnits" version="01.00.01" alias="rru" />
+
+    <ECCustomAttributes>
+        <ProductionStatus xmlns="CoreCustomAttributes.01.00.03">
+            <SupportedUse>NotForProduction</SupportedUse>
+        </ProductionStatus>
+        <SchemaLayerInfo xmlns="BisCustomAttributes.01.00.00">
+            <Value>Application</Value>
+        </SchemaLayerInfo>
+    </ECCustomAttributes>
+
+    <ECEntityClass typeName="Draft" displayLabel="Draft">
+        <BaseClass>bis:GraphicalElement3d</BaseClass>
+    </ECEntityClass>
+
+    <ECEntityClass typeName="Point" displayLabel="Point">
+        <BaseClass>Draft</BaseClass>
+        <ECProperty propertyName="X" typeName="double" kindOfQuantity="rru:LENGTH" displayLabel="Easting"/>
```

Comment:
I'm not sure if "length" is the right quantity for coordinates.

@diegoalexdiaz this has come up a couple of times - what do you think?